lib/composer/composer/ClassLoader.php

```diff
@@ -377,7 +377,7 @@ private function findFileWithExtension($class, $ext)
             $subPath = $class;
             while (false !== $lastPos = strrpos($subPath, '\\')) {
                 $subPath = substr($subPath, 0, $lastPos);
-                $search = $subPath . '\\';
+                $search = $subPath.'\\';
```

Comment:
Not sure which formatting is correct and consistent with the rest of the code

Reply:
this code is autogenerated by composer, so it shouldn't be edited manually :)